### DISCUSSION TITLE
Help with locality

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#5244] [Announcements]
I have some data that includes the locality "Uc Berkeley Campus". So I'm thinking I can probably use some great, georeferenced locality that already exists in Arctos, but the one that looks useful doesn't have coordinates.

![image](https://user-images.githubusercontent.com/5725767/193065649-a2905abe-70a3-4bb0-86b7-676eb621f791.png)

Am I searching for the wrong thing? Is there a georeferenced locality for the UC Berkeley Campus? @mkoo @atrox10 @ccicero 

[downloadCollectingEvent_2953655091135221.csv](https://github.com/ArctosDB/arctos/files/9675816/downloadCollectingEvent_2953655091135221.csv)

**Comment by Jegelewicz** [comment]
Also, I always hear about our "gazetteer" and being able to use existing "good" localities, but I don't know how to do this. Anyone have advice? I have a list of about 1,800 localities, some of which I guess already exist in Arctos - how can I match what I have with what's already there?

**Comment by ewommack** [comment]
> Also, I always hear about our "gazetteer" and being able to use existing "good" localities, but I don't know how to do this.

I'd also love to know how to use!

**Comment by dustymc** [comment]
<img width="712" alt="Screen Shot 2022-09-29 at 9 54 33 AM" src="https://user-images.githubusercontent.com/5720791/193092548-08768e80-39b7-429b-8dcf-52d24155744d.png">

oh my!

<img width="703" alt="Screen Shot 2022-09-29 at 9 56 41 AM" src="https://user-images.githubusercontent.com/5720791/193092572-d5ac3bc6-f0c7-47b1-aa1c-83d195c5b823.png">

Showing 1 to 100 of 962 entries

OH MY!

Not very useful, but maybe a good example of why I want more normalization rather than less - and maybe we should revist the 'max max error' thing.


<img width="706" alt="Screen Shot 2022-09-29 at 9 59 00 AM" src="https://user-images.githubusercontent.com/5720791/193093179-d1a9e062-1428-4698-9f5f-505449f053d3.png">

maps nicer, but still...

Showing 1 to 100 of 464 entries

I don't seem to have any help, but perhaps this is all indicative of us simply doing it wrong - eg these...


<img width="128" alt="Screen Shot 2022-09-29 at 10 42 58 AM" src="https://user-images.githubusercontent.com/5720791/193104429-3c4ab387-1381-474c-920e-6d2a67f6a2d9.png">

differ by small bits of coordinate+error and not-very-informative metadata

<img width="713" alt="Screen Shot 2022-09-29 at 10 43 19 AM" src="https://user-images.githubusercontent.com/5720791/193104528-fa0e6561-0a97-4a2f-92a3-ca91eaa359ac.png">

and even these

<img width="138" alt="Screen Shot 2022-09-29 at 10 54 29 AM" src="https://user-images.githubusercontent.com/5720791/193106403-538fb428-0016-4998-bc97-34001c9b8891.png">

might not really be functionally distinct - is there some plausible question to which the answer might depend on that eighth of a mile being faithfully represented in the "less-verbatim" structure?

See also https://github.com/ArctosDB/newsletter/issues/171 - it's a bit cleaner, 25 localities holding 273 records in the random box I drew on a map, but still, three-ish of them are about a golf course, 6 of them mention research park, junction of Colorow Rd. and Wakara Way twice and 301 Wakara Way three times, etc. Fixing whatever proportion of those 250+ records that needs it involves something less than 25 clicks, which isn't too bad, but it's still perhaps double or triple what's really necessary to form what from here looks like an entirely adequate representation of the descriptions.

My current views are obviously being influenced by the geography discussions, and there it's looking more and more like the choice is between being slightly coarser-grained, or having so many so precise things that we can't possible manage them, much less form defensible relationships between them. ("This georeference shall use that geography which it does not intersect" occurs thousands of times.) Do we perhaps need to shift a bit more of the data towards 'verbatim' structures, and increase the steepness of the normalization curve as we move "formal-ward"? 

>always hear about our "gazetteer"

That's the first time I've heard about it....

You might try https://localityservice.uc.r.appspot.com/ - I use it for back-end stuff, but it's not Arctos.

**Comment by Jegelewicz** [comment]
I don't think this should be closed. None of my questions were answered!

1. Is there a georeferenced locality for the UC Berkeley Campus? @mkoo @atrox10 @ccicero

[downloadCollectingEvent_2953655091135221.csv](https://github.com/ArctosDB/arctos/files/9675816/downloadCollectingEvent_2953655091135221.csv)

2. I always hear about our "gazetteer" and being able to use existing "good" localities, but I don't know how to do this. Anyone have advice? I have a list of about 1,800 localities, some of which I guess already exist in Arctos - how can I match what I have with what's already there?
